README.md

```diff
@@ -1,208 +1,256 @@
+Download your Apple Push Notification service SSL certificate and add it to your Keychain.
 
-### Step 1 - Login to Mendix home
-Open up home.mendix.com and navigate to the project that you wish to build the app for. Once on the project wall for your application click on the publish section.
+This certificate needs to be converted into `.p12` format. If you don’t know how to do it, please refer to [this page]  (https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/MaintainingCertificates/MaintainingCertificates.html).
 
-<img src="assets/images/step1.png"/>
+### Step 6 - Configure APNS on your application
 
-### Step 2 - App Identifier
-Once in the publish section you must enter an app identifier for your application. This is important when setting up an IOS app. Refer to the section about creating an app identifier to find out more information about creating an app identifier and IOS certificate.
+For the last step you need to configure the APNs setting within your application. This can be done by logging into your application as a user with Administrator role and opening the Apple Administration page that was set up in step 7 of the Implementation Guide.
 
-<img src="assets/images/step2.png"/>
+For this purpose you need to:
+-	Add your Apple Push Notification service SSL certificate in `.p12` format
+-	Add server url and port. For sandbox it is `gateway.sandbox.push.apple.com:2195` and `gateway.push.apple.com:2195` for production.
+-	Add feedback url and port. For sandbox it is `feedback.sandbox.push.apple.com:2196` and `feedback.push.apple.com:2196` for production.
 
-### Step 3 - Select Devices
-Select the devices that you want to deploy your app to and upload splash screen images for the devices you have selected. 
+## Setting up Google Cloud Messaging Server
+
+In order to send google push notifications from this module you need to have set up a google account with google cloud messaging enabled.
+To do so follow these steps to get registered for Google cloud messaging and enter the details into the Mendix screens.
 
-Press the button publish to appstore and you will be asked whether you want to build in the cloud or do it yourself. Choose do it yourself and press the Download Phonegap Build Package.
+### Step 1 - Login to developers console
+
+Open up the Google [developers console] (https://console.developers.google.com) and login with your Google id.
+<img src="assets/images/gcm-step1.PNG"/>
 
+### Step 2 - Create project
 
-<img src="assets/images/step3.png"/>
+Click new project and fill in the project name and project ID for your application. Then click create. Once created you will see a project number at the top of the screen next to the project ID. Take note of this ID because you will need it later on for our sender ID.
```

Comment:
This is not true. After creating a project I did not find the project number at the .